uint8 joint_id
float64 angle
